docs/semgrep-secrets/historical-scanning.md

```diff
@@ -0,0 +1,114 @@
+---
+slug: historical-scanning
+append_help_link: true
+title: Scan your git history
+hide_title: true
+description: Detect valid, leaked secrets in previous git commits through a historical scan.
+tags:
+  - Semgrep Secrets
+  - Team & Enterprise tier
+---
+
+# Scan your git history
+
+Detect valid, leaked secrets in previous git commits through a **historical scan**.
+
+You can perform one-time historical scans or enable historical scanning for all Secrets scans. Detecting valid secrets in your git history is a first step to reducing your repository's attack surface.
+
+You can run historical scans in the CLI and in your Semgrep deployment, which enables you to track and triage these secrets.
+
+:::note Feature maturity
+- This feature is in **public beta**. See [Limitations](#limitations) for more information.
+- All Semgrep Secrets customers can enable this feature.
+- Please leave feedback by either reaching out to your technical account manager (TAM) or through the **<i class="fa-solid fa-bullhorn"></i> Feedback** form in Semgrep Cloud Platform's navigation bar.
+:::
+
+## Run historical scans
+
+You can enable historical scanning for all of your future secret scans or run a dedicated CI job for one-time scans. By default, historical scans display **valid, leaked secrets** to ensure a high true positive rate.
```

Comment:
historical scanning just runs for full scans, not all scans

Reply:
It could also be worth explicitly mentioning that we do not display secrets without validators, and things like validation errors are ommitted